src/components/field/GlobalProgramDisableReferral.js

```diff
@@ -0,0 +1,39 @@
+import i18n from '@dhis2/d2-i18n'
+import cx from 'classnames'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { CheckboxField } from './CheckboxField'
+import styles from './Field.module.css'
+
+const CODE = 'disableReferral'
```

Comment:
NIT: `const CODE` is also declared and used in `DisableReferral.js`. Does it make sense to move it to some common constants file?